bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -221,7 +221,7 @@ impl From<matrix_sdk::TransmissionProgress> for TransmissionProgress {
 pub struct Client {
     pub(crate) inner: AsyncRuntimeDropped<MatrixClient>,
     delegate: RwLock<Option<Arc<dyn ClientDelegate>>>,
-    utd_hook_manager: Option<Arc<UtdHookManager>>,
+    utd_hook: Option<Arc<UtdHookManager>>,
```

Comment:
`utd_hook_manager` was the right name (it's a thin wrapper over the actual UTD hook provided by the user), can you keep it?